app/assets/javascripts/pdfjs_viewer/viewer.js

```diff
@@ -941,7 +941,7 @@ var DEFAULT_SCALE_DELTA = 1.1;
 var DISABLE_AUTO_FETCH_LOADING_BAR_TIMEOUT = 5000;
 function configure(PDFJS) {
   PDFJS.imageResourcesPath = '/pdfjs/web/images/';
-  PDFJS.workerSrc = '/pdfjs/web/pdf.worker.js';
+  PDFJS.workerSrc = '/pdfjs/web/pdf.worker.js?version=1.10.100';
```

Comment:
is any other resource loaded like this? would be worth checking and adding the query string param there, too.

Reply:
The images and cmaps are, but they're less volatile and not on the critical path. As they're defined as directories I'd need to find how they're actually used everywhere where they get used.